tests/generic_tests.robot

```diff
@@ -5,10 +5,10 @@ Metadata            Version    0.0.1
 Library             SeleniumLibrary
 Resource            ../resources/keywords/deploy_instance_dev.resource
 Resource            ../resources/keywords/import_provider_account.resource
-Resource            ../resources/keywords/suite_and_test_teardown.resource
+
```

Comment:
If you remove this resource, wouldn't there be a problem running the TearDown?

Reply:
The tear suite teardown is removed for each test suite and has been added to login.resource at line number 9, this is done to keep minimum declaration of static resources at test suite level. Every test suite utilizes login.resource, and hence will inherently have access to suite_and_test_teardown.resource